docs/operator-guides/using-custom-connectors.md

```diff
@@ -0,0 +1,111 @@
+2. You want to build a connector for an **internal** source or destination (private API) specific to your organization. This connector has no good reason to be exposed to the community.
+
+This guide focuses on the second approach and assumes the following:
+* You followed our other guides and tutorials about connector developments.
+* You finished your connector development, running it locally on an Airbyte development instance.
+* You want to deploy this connector to a production Airbyte instance running on a VM with docker-compose or on a Kubernetes cluster.
+
+If you prefer video tutorials, [we recorded a demo about uploading connectors images to a GCP Artifact Registry](https://www.youtube.com/watch?v=4YF20PODv30&ab_channel=Airbyte).
+
+## 1. Create a private Docker registry
+Airbyte needs to pull its Docker images from a remote Docker registry to consume a connector.
+You should host your custom connectors image on a private Docker registry. 
+Here are some resources to create a private Docker registry, in case your organization does not already have one:
+
+| Cloud provider | Service name                | Documentation                                                                                                                                                                                                                                                                              |
+|----------------|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Google Cloud   | Artifact Registry           | [Quickstart](https://cloud.google.com/artifact-registry/docs/docker/quickstart)|
+| AWS            | Amazon ECR                  | [Getting started with Amazon ECR](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-console.html)|
+| Azure          | Container Registry          | [Quickstart](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal#:~:text=Azure%20Container%20Registry%20is%20a,container%20images%20and%20related%20artifacts.&text=Then%2C%20use%20Docker%20commands%20to,the%20image%20from%20your%20registry.)|
+| DockerHub      | Repositories                | [DockerHub Quickstart](https://docs.docker.com/docker-hub/)|
+| Self hosted    | Open-source Docker Registry | [Deploy a registry server](https://docs.docker.com/registry/deploying/)|
+
+## 2. Authenticate to your private Docker registry
+To push and pull images to your private Docker registry, you need to authenticate to it:
```

Comment:
Maybe use Private Container Registry?

Reply:
I think the common naming is _Docker registry,_ and in our case we want it to be private 😄 . _Container registry_  is a cloud service name.